addon/ng2/blueprints/ng2/index.js

```diff
@@ -3,23 +3,34 @@ const stringUtils = require('ember-cli-string-utils');
 
 module.exports = {
   description: '',
+  
+  availableOptions: [
+    { name: 'source-dir', type: String, default: 'src', aliases: ['sd'] },
+    { name: 'prefix', type: String, default: 'app', aliases: ['p'] }
+  ],
 
   locals: function(options) {
     //TODO: pull value from config
     this.styleExt = 'css';
     this.version = require(path.resolve(__dirname, '..', '..', '..', '..', 'package.json')).version;
-    
+    const refToTypings = options.sourceDir.split(path.sep).map(() => '..').join('/');
```

Comment:
`path.normalize` this too. Also, instead of splitting by `path.sep`, could you `split(/[\\\/]/g)`?